src/test/ui/consts/const_let_eq_float.rs

```diff
@@ -0,0 +1,277 @@
+#![feature(const_let, const_fn)]
```

Comment:
You forgot to `--bless` and to include `.stderr`

Reply:
I did not, that test is passing, as it should (`const_fn` feature gate allows this). Removing the `const_fn` feature gate fails way before we ever see a float related error.

Reply:
oh... but the point of the test was to ensure that `+=` et. al don't work under just `#![feature(const_let)]`.

Reply:
That's impossible to test by the very nature of min_const_fn

Reply:
How come? In `min_const_fn.rs` we have:
```rust
const fn foo19(f: f32) -> f32 { f * 2.0 }
//~^ ERROR only int, `bool` and `char` operations are stable in const fn
```
so why can we not have:
```rust
const fn bar() { let x: f32 = 1.0; x+= 2.0; }
//~^ ERROR only int, `bool` and `char` operations are stable in const fn
```

Reply:
because if we allowed let bindings in min const fn, the standard library's *stable* const fns might accidentally use let bindings without let bindings being stable.

Reply:
Conclusion: we'll need to keep these tests as-in now; once we make `const_let` stable then the floating point tests should be compile-fail.